book/features/external_toc.md

```diff
@@ -0,0 +1,69 @@
+(teachbooks-package)=
+# Sharing content between books in table of contents
+
+```{admonition} User types
+:class: tip
+This page is useful for user type 3-5.
+```
+
+{bdg-secondary}`Python Package`
+
+## Introduction
+When creating books, you might want to reuse material from other people or from other books you made. In some cases you might even want to have the exact same material into your book. You could do so by manually copying material over. However, whenever the source material is updated, you have to do that again. As an alternative, you can use the underlying git system to refer to the source file directly. This allows you to pick a specific version, or keep the most up-to-date version of it. This pages explains how to do so directly in the table of contents using the TeachBooks package.
+
+Previously, this was implemented using [submodules] (../external/Nested-Books/README.md), but this implementation was more difficult to use. However, the approach using submodules allows to embed pages which: refer to content in the `_static` folder of the other book, include local images referenced with raw-HTML code, are part of a private book. Furthermore, if you'd like to be notified on updates of the other book, the approach using submodules allows notifications in any case, while the approach described on this page only allows for notifications by 'watching' the other book repository and requiring the other repo owner to release new version with 'releases'.
+
+## What does it do?
+
+This functionality, part of the [TeachBooks package](https://github.com/TeachBooks/TeachBooks) allows you to refer to `.ipynb` and `.md` files stored on public repositories. These files are then handles as if they were normal files in your book, leading to a the file as a page in your built book. To prevent issues it warns you if there's a mismatch in `requirements.txt`, combines `.bib`-files, merges plugins in `_config.yml` and checks licenses.
```

Comment:
```suggestion
This functionality, part of the [TeachBooks package](https://github.com/TeachBooks/TeachBooks) allows you to refer to `.ipynb`, `.md` and `.rst` files stored on public repositories. These files are then handles as if they were normal files in your book, leading to a the file as a page in your built book. To prevent issues it validates that the external content has a permissive license, automatically combines any `reference.bib` files, and warns you if there's a mismatch in `requirements.txt` or with plugins in `_config.yml`.
```

We should also link to the docs on https://teachbooks.io/TeachBooks/ once the PR is merged.